app/controllers/development_controller.rb

```diff
@@ -274,6 +281,15 @@ def test_discussion
     @test_discussion
   end
 
+  def test_discussions
+    100.times do
+      Discussion.create!(title: Faker::Company.bs,
```

Comment:
If we end up using this in more than one test, we should look at the active-record-import gem (https://github.com/zdennis/activerecord-import), because it'll start slowing down our e2ez really quickly.